src/main/java/org/jabref/model/cleanup/FieldFormatterCleanup.java

```diff
@@ -71,28 +69,18 @@ public FieldFormatterCleanup(Field field, Formatter formatter) {
     }
 
     private List<FieldChange> cleanupAllFields(BibEntry entry) {
-        List<FieldChange> fieldChanges = new ArrayList<>();
-
-        for (Field fieldKey : entry.getFields()) {
-            if (!fieldKey.equals(InternalField.KEY_FIELD)) {
-                fieldChanges.addAll(cleanupSingleField(fieldKey, entry));
-            }
-        }
-
-        return fieldChanges;
+        return entry.getFields().stream()
+             .filter(field -> !field.equals(InternalField.KEY_FIELD))
+             .flatMap(field -> cleanupSingleField(field, entry).stream())
+             .collect(Collectors.toList());
     }
 
     private List<FieldChange> cleanupAllTextFields(BibEntry entry) {
-        List<FieldChange> fieldChanges = new ArrayList<>();
-        Set<Field> fields = new HashSet<>(entry.getFields());
-        fields.removeAll(FieldFactory.getNotTextFieldNames());
-        for (Field fieldKey : fields) {
-            if (!fieldKey.equals(InternalField.KEY_FIELD)) {
-                fieldChanges.addAll(cleanupSingleField(fieldKey, entry));
-            }
-        }
-
-        return fieldChanges;
+        return entry.getFields().stream()
```

Comment:
Not sure about our convention, but I don't think we should strive for converting every for loop now to streams just for the sake of it. (Maybe it's just my personal bias and I usually have a harder time to unravel a lot of filter/map combinations that some old-fanished for loop with ifs...)

Reply:
Since I use streams very often, I find the old code more hard to read. Maybe, we need to fix our style there. Or let it go that each dev can do however he wants (with new code; and don't touch old code)